app/src/main/java/app/revanced/manager/ui/component/tooltip/TooltipIconButton.kt

```diff
@@ -0,0 +1,98 @@
+package app.revanced.manager.ui.component.tooltip
+
+import androidx.annotation.StringRes
+import androidx.compose.foundation.interaction.MutableInteractionSource
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.IconButtonColors
+import androidx.compose.material3.IconButtonDefaults
+import androidx.compose.material3.TooltipDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.hapticfeedback.HapticFeedbackType
+import androidx.compose.ui.window.PopupPositionProvider
+
+/**
+ * [IconButton] with tooltip-specific params.
+ *
+ * @param tooltip [String] text to show in a tooltip.
+ * @param positionProvider [PopupPositionProvider] Anchor point for the tooltip.
+ * @param haptic Whether to perform haptic feedback when the tooltip shown.
+ * @param hapticFeedbackType The type of haptic feedback to perform.
+ *
+ * @see [IconButton]
+ */
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+fun TooltipIconButton(
+    onClick: () -> Unit,
+    modifier: Modifier = Modifier,
+    enabled: Boolean = true,
+    colors: IconButtonColors = IconButtonDefaults.iconButtonColors(),
+    interactionSource: MutableInteractionSource = remember { MutableInteractionSource() },
+    tooltip: String,
+    positionProvider: PopupPositionProvider = TooltipDefaults.rememberPlainTooltipPositionProvider(),
+    haptic: Boolean = true,
+    hapticFeedbackType: HapticFeedbackType = HapticFeedbackType.LongPress,
+    content: @Composable (() -> Unit),
+) {
+    TooltipWrap(
+        modifier = modifier,
```

Comment:
If you need to pass modifiers to `TooltipWrap`, use a separate parameter, otherwise remove it. This also applies for the other components.